### ISSUE TITLE
luci visual improvement

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#2406]
No idea how much this is doable, but: the luci model currently used could use a plasma ball forming as the player charges the weapon.

**Comment by necessarily-equal** [comment]
Agreed, see also https://github.com/Unvanquished/Unvanquished/issues/2391
